products/sle/main.pm

```diff
@@ -74,6 +74,22 @@ sub is_bridged_networking {
     return $ret;
 }
 
+sub default_desktop {
+    my $defdesktop = 0;
```

Comment:
yeah, I went for this approach because I wanted to reflect the theoretical technical possibility that there might not be any default.

In which case, any compare on the return of default_desktop must fail, which would then trigger the appropriate test logic to set the default.

The idea of 'no default' is being kicked around for Tumbleweed, so not relevant for SLE or here, but it tipped my hand when it came to the matter of taste.

Also it helped debugging when I wasn't hitting the globs properly ;)